app/models/target.rb

```diff
@@ -0,0 +1,8 @@
+class Target < ApplicationRecord
+  belongs_to :user  
+  #validates :user_id, presence: true
+  validates :title, length: { maximum: 140 },uniqueness: { case_sensitive: false }
+  validates_presence_of :title, :topic, :length, :latitude, :longitude
+
+  enum topic: [:football, :travel, :politics, :art, :dating, :music, :movies, :series, :food]
```

Comment:
Also, just a minor comment. 
I think in this case this is ok but most of the times it would be better to declare the enum with explicit values like this:

`enum topic: { football: 0, travel: 1.... etc } `

Reply:
Also, using numbers to reference each enum is a good practice:
```
enum topic: {
    football: 0,
    travel: 1
  }
```